src/redux/redux-store.js

```diff
@@ -2,21 +2,44 @@ import { createStore, applyMiddleware } from 'redux'
 import createSagaMiddleware from 'redux-saga'
 import { composeWithDevTools } from 'redux-devtools-extension'
 import createDebounce from 'redux-debounced'
+import { persistStore, persistReducer } from 'redux-persist'
+import autoMergeLevel2 from 'redux-persist/lib/stateReconciler/autoMergeLevel2'
+import {
+  createWhitelistFilter,
+  createBlacklistFilter,
+} from 'redux-persist-transform-filter'
+import storage from 'redux-persist/lib/storage'
 
 import rootSaga from './saga'
 import rootReducers from './reducers'
 
-const initialState = {}
 const sagaMiddleware = createSagaMiddleware()
 const debounceMiddleware = createDebounce()
 const middleware = [sagaMiddleware, debounceMiddleware]
 
+const persistConfig = {
+  key: 'root',
+  storage,
+  stateReconciler: autoMergeLevel2,
+  transforms: [
+    createWhitelistFilter('imagesGalleryPage', [
+      'searchText',
+      'yearStart',
+      'yearEnd',
+    ]),
+    createBlacklistFilter('imgDailyPage', ['image']),
+  ],
+}
+
+const persistedReducer = persistReducer(persistConfig, rootReducers)
+
 const store = createStore(
-  rootReducers,
```

Comment:
should you not pass only the persisted reducer to the createStore method?

Reply:
this line is empty